pysass.cpp

```diff
@@ -404,6 +435,116 @@ static void _add_custom_functions(
     sass_option_set_c_functions(options, fn_list);
 }
 
+Sass_Import_List _call_py_importer_f(
+        const char* path,
+        Sass_Importer_Entry cb,
+        struct Sass_Compiler* comp
+) {
+    PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
+    PyObject* py_path = PyUnicode_FromString(path);
```

Comment:
Indeed.  That also goes for the format string I switched the `ParseTuple` statements to that is cross-verson consistent.  It also does UTF-8 conversion, which, I feel, is a much more sane default.  ;)